src/components/Comments/DetailDomainScreenCommentItem.js

```diff
@@ -187,7 +183,7 @@ const styles = StyleSheet.create({
   btnReplyText: {
     fontFamily: fonts.inter[400],
     fontSize: 13,
-    color: COLORS.lightgrey,
+    color: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the reply text button has been changed. Verify that this new color fits into the overall design scheme and does not negatively affect usability or accessibility.
```diff
- color: COLORS.lightgrey,
+ color: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->